### DISCUSSION TITLE
Is it possible/recommended use pixijs to create a video editor?

### DISCUSSION THREAD

**Discussion by Zzzen** [pixijs/pixijs#9244] [Q&A] [answered]
https://pixijs.io/examples/#/sprite/video.js

It's easy to play video with pixijs, but I cannot find any docs about controlling the state like play/pause/seek.

**Comment by SuperSodaSea** [comment, marked as answer]
Some related discussions: #9083, #9212

You can use `texture.baseTexture.resource.source` to access the underlying [`HTMLVideoElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLVideoElement), use its methods like [`play()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/play), [`pause()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/pause), and seek it by setting its [`currentTime`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/currentTime) property.
